app/uk/gov/nationalarchives/omega/editorial/services/jms/ResponseBuilder.scala

```diff
@@ -44,26 +44,50 @@ class ResponseBuilder[F[_] : ResponseBuilder.ME : Logger] {
 
   def createResponseText(jmsMessage: JmsMessage): F[String] =
     jmsMessage.getStringProperty(sidHeaderKey) match {
-      case Some(sidValue) if sidValue == sid1 =>
-        logger.info(s"got a message with sid header '$sidValue', trying to parse payload") *>
-          getEditSet(jmsMessage).flatMap { editSet =>
-            logger.info("parsed payload, creating edit set response") *>
-              me.pure(Json.toJson(editSet).toString)
-          }
-
+      case Some(sidValue) if sidValue == SID.GetEditSet =>
+        handleRequestForGettingAnEditSet(jmsMessage, sidValue)
+      case Some(sidValue) if sidValue == ResponseBuilder.SID.GetEditSetRecord =>
```

Comment:
Small point but the `ResponseBuilder.` prefix is not needed.

Reply:
Will change.

Reply:
Changed.

Reply:
Marking resolved, as you've approved, @felixgb .